contrib/tools/stacks-relay/src/state.rs

```diff
@@ -0,0 +1,50 @@
+use std::collections::HashMap;
+
+#[derive(Default)]
+pub struct State {
+    /// The value for this map is an index for the last read message for this node.
+    highwaters: HashMap<String, usize>,
+    queue: Vec<String>,
```

Comment:
I guess we're okay with this growing forever as this is just for the alpha testnet.